test/recipes/controls/aws_parallelcluster_install/directories_spec.rb

```diff
@@ -0,0 +1,21 @@
+control 'setup_dirs' do
```

Comment:
Thought: wouldn't something like "ParallelCluster directories exist" be more appropriate? Would it be possible to use a string like this in the kitchen controls list? If now, we can by any means use "pcluster_directories_exist".
I would like to define this ASAP, so that we don't have to refactor lots of stuff later.
What do you think?